{{ cookiecutter.municipality_slug }}/python/example_scraper.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import random
+from datetime import datetime
+from typing import List
+
+from cdp_backend.pipeline.ingestion_models import (
+    Body,
+    EventIngestionModel,
+    EventMinutesItem,
+    Matter,
+    MinutesItem,
+    Person,
+    Role,
+    Seat,
+    Session,
+    SupportingFile,
+    Vote,
+)
+
+###############################################################################
+
+
+def get_example_events(
+    n: int = 10, is_minimal: bool = False
+) -> List[EventIngestionModel]:
+    return [_get_example_event(is_minimal) for _ in range(n)]
+
+
+###############################################################################
+
+
+FAKE_COUNCIL_SEAT_NUM = 10
+
+
+def _get_example_event(is_minimal: bool) -> EventIngestionModel:
+    "Create a fake example event data"
+    # Create a body for the event
+    body = Body(
+        name=f"Example Committee {random.randint(1, 100)}",
+        description="Example Description",
+    )
+    # Create sessions for the event
+    sessions = [
+        Session(
+            session_datetime=datetime.utcnow(),
+            video_uri="https://youtu.be/UjieH-NrOe8",
+        )
```

Comment:
Can we have a random of 80% of the time the session _has_ a closed caption file?